apps/stress-test/src/shared/react/TestMount.tsx

```diff
@@ -27,7 +27,7 @@ const debouncedOnRender: DebouncedOnRender = () => {
           end: performance.now(),
         });
       });
-    }, 250);
+    }, 250) as unknown) as number;
```

Comment:
Fixes this error:
![image](https://user-images.githubusercontent.com/8649804/195950561-0f42a0f6-d3fa-42ad-bf9e-e180d4cb65f3.png)